operations/business-operations/analytics/metrics-definitions.md

```diff
@@ -216,19 +216,43 @@ As soon as one session ends, there is then an opportunity to start a new session
+
+### Quarterly Trailing NPS
+
+[Quarterly Trailing NPS](https://mattermost.looker.com/dashboards/147) represents the NPS score computed based on survey submissions received in the last 90 days. We target a 90 day window of NPS responses because it:
+
+1. Encompasses responses across all server versions that are currently in use by customers (with surveys enabled), meaning it is representative of the state of user experience that customers are facing today.
+2. Ensures we have a [statistically significant sample size](https://www.checkmarket.com/sample-size-calculator/) representing the [server versions currently in use](https://mattermost.looker.com/looks/203?toggle=dat,det,pik).
+3. Helps us capture week-to-week variations in NPS while being less volatile than NPS by server version given the larger sample size
+
+
+### NPS by Server Version
+
+[NPS by Server Version](https://mattermost.looker.com/dashboards/147) represents the NPS score computed based on survey submissions on a specific server version. While Quarterly Trailing NPS provides a representation of the state of user experience that our customers are facing, NPS by Server Version provides a representation of the user experience offered by the product in particular releases. As such, it's used heavily by the PM team to track the success of particular product team initiatives as they ship.
+
+NPS by Server Version is a lagging metric since we need to collect a statistically significant sample size before reporting NPS for a server version. Based on historical data, ~2000 unique user responses (~1.5-2 months post-ship) are required for the NPS of a particular server version to stabilize. 
+
+NPS by Server Version can be volatile since it can be affected by the upgrade cadence of heavy usage servers. We typically see a surge in responses when surveys are triggered (21 days after server upgrade) for servers with large DAU, which can impact the NPS trend for that server version. Quarterly trailing NPS is not as affected by this since the sample size is larger and responses are submitted on various server versions. 
+
+### Written Response Feedback
+
+Users can optionally submit written responses to the NPS survey. These responses are reviewed and categorized by the PM team to allow us to [stack rank](https://docs.google.com/spreadsheets/d/1cWqg_djHd_Xmq7cuG0YZj-lKmafgCHU_dU4KfRkClrY/edit#gid=0) certain initiatives for roadmap planning based on their impact to NPS, reach and effort to design, implement and test.
+
+To determine impact to NPS, we can:
+1. Segment the reponses based on written feedback from users who are detractors or passives (See the NPS Feedback section of the [NPS dashboard](https://mattermost.looker.com/dashboards/147))
+2. Segment the responses based on the feedback category to analyze [what requested product enhancements equate to the lowest NPS scores from users](https://docs.google.com/spreadsheets/d/1g6Uv5BHm54TEnXfUPBFzj_IYysqxUMt3ifgdemRPAKU/edit#gid=1937727547).
```

Comment:
Same with these links - are we going to continue to use the same link over time? If not, consider either cutting or calling out that this is an example so the handbook is less likely to become dated.